{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2021-45928",
			"ASSIGNER": "cve@mitre.org"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=36456",
					"name": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=36456",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/libjxl/libjxl/pull/365",
					"name": "https://github.com/libjxl/libjxl/pull/365",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/google/oss-fuzz-vulns/blob/main/vulns/libvips/OSV-2021-1055.yaml",
					"name": "https://github.com/google/oss-fuzz-vulns/blob/main/vulns/libvips/OSV-2021-1055.yaml",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/libjxl/libjxl/issues/360",
					"name": "https://github.com/libjxl/libjxl/issues/360",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/libjxl/libjxl/compare/v0.5...v0.6",
					"name": "https://github.com/libjxl/libjxl/compare/v0.5...v0.6",
					"refsource": "MISC",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "libjxl before 0.6, as used in libvips 8.11 through 8.11.2 and other products, has an out-of-bounds write in jxl::ModularFrameDecoder::DecodeGroup (called from jxl::FrameDecoder::ProcessACGroup and jxl::ThreadPool::RunCallState<jxl::FrameDecoder::ProcessSections)."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2022-01-01T01:15Z",
	"lastModifiedDate": "2022-01-01T01:15Z"
}